src/Markup/Avalonia.Markup.Xaml.Loader/CompilerExtensions/AvaloniaXamlDiagnosticCodes.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using XamlX;
+
+namespace Avalonia.Markup.Xaml.XamlIl.CompilerExtensions;
+
+internal static class AvaloniaXamlDiagnosticCodes
```

Comment:
Todo: revisit these values, some of them are unused
Do we need to have unique diagnostic codes per each error? Currently, it's grouped by area and ability to lower severity, i.e. two different selector parse errors will have the same code - AVLN2201 - right now.